service/create/service.go

```diff
@@ -896,12 +880,50 @@ func (s *Service) onAdd(obj interface{}) {
 		instanceProfileName: policy.GetName(),
 		prefix:              prefixWorker,
 	}
-	if _, err := s.createLaunchConfiguration(lcInput); err != nil {
+
+	lcCreated, err := s.createLaunchConfiguration(lcInput)
+	if err != nil {
+		s.logger.Log("error", errgo.Details(err))
+		return
+	}
+
+	if lcCreated {
+		s.logger.Log("info", fmt.Sprintf("created worker launch config"))
+	} else {
+		s.logger.Log("info", fmt.Sprintf("launch config %s already exists, reusing", cluster.Name))
+	}
+
+	workersLCName, err := launchConfigurationName(cluster, "worker", workersSecurityGroupID)
```

Comment:
Agreed this should use the constant please add to the issue. No strong opinion on the refactoring.